src/App.tsx

```diff
@@ -1,8 +1,16 @@
-import { Loader } from './components/Loader';
-
+import {
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import React from 'react';
 import './App.scss';
+import { HomePage } from './components/HomePage/HomePage';
+import { PageNotFound } from './components/PageNotFound/PageNotFound';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { HeaderNavLink } from './components/HeaderNavLink/HeaderNavLink';
 
-export const App = () => (
+export const App: React.FC = () => (
   <div data-cy="app">
     <nav
```

Comment:
Put nav to a Navigation component and create a Layout component that will serve as a wrapper for all your routes. Within this Layout, we'll include a Navigation component and a main section to wrap the content. To achieve this, we can use the Outlet component from react-router-dom.

```
<Routes>
  <Route path="/" element={<Layout />}>
    .. homepage
    ...tabs
    ...
 </Routes>   
```